internal/session/repository_session.go

```diff
@@ -367,6 +376,7 @@ func (r *Repository) TerminateCompletedSessions(ctx context.Context) (int, error
 // and returns it, along with it's states.  If the authorization fails, it
 // an error of ErrInvalidStateForOperation.
```

Comment:
I wish we were not raising gRPC status errors in the domain... such a bad pattern for the domain layer to be protocol aware.